app/services/digest_initiator_service.rb

```diff
@@ -8,20 +8,50 @@ def self.call(*args)
   end
 
   def call
+    digest_run = create_digest_run
+    return if digest_run.nil?
+
+    subscribers = DigestRunSubscriberQuery.call(digest_run: digest_run)
+
+    digest_run_subscriber_params = build_digest_run_subscriber_params(
+      digest_run,
+      subscribers
+    )
+
+    enqueue_jobs(
+      DigestRunSubscriber.import!(digest_run_subscriber_params).ids
```

Comment:
I wonder if we should make this line more of an explicit step rather than as part of arguments to a call, might make it slightly clearer.